### DISCUSSION TITLE
Mermaid is not rendered in sub-page of projects

### DISCUSSION THREAD

**Discussion by Keith-CY** [community/community#18596] [Projects and Issues]
In sub-page
![image](https://user-images.githubusercontent.com/7271329/173627552-4cabcc26-cba3-46b8-8f4b-868dc0a77a61.png)

In new tab
![image](https://user-images.githubusercontent.com/7271329/173627696-57bea888-c066-491e-979f-14a124bd7008.png)

**Comment by rileybroughten** [comment]
👋 Hi @Keith-CY, thank you for the feedback! This has been raised with the team and we will try and prioritize it against our backlog and give you an update once it is resolved 👍

**Comment by iansan5653** [comment]
👋 Hello! I think this may have been resolved by some related work. Can you confirm that the issue is fixed on your end?

**Reply by Keith-CY** [reply]
Thanks for the work, they flash several times but are rendered properly at the end.

https://user-images.githubusercontent.com/7271329/187040684-8e02b433-d4ee-429e-a08d-79bd07db0733.mov

**Comment by chrislujan** [comment]
I'm experiencing the initial problem, an issue renders some example mermaid code just fine, but in the project view, I get mixed results but never a successful render:
````
```mermaid
sequenceDiagram
    participant dotcom
    participant iframe
    participant viewscreen
    dotcom->>iframe: loads html w/ iframe url
    iframe->>viewscreen: request template
    viewscreen->>iframe: html & javascript
    iframe->>dotcom: iframe ready
    dotcom->>iframe: set mermaid data on iframe
    iframe->>iframe: render mermaid
```
````
results in:
![Screen Shot 2022-10-26 at 3 39 37 PM](https://user-images.githubusercontent.com/19271145/198152006-23b8c527-bdd4-48d5-bbe6-b985abbf277b.png)
then
![Screen Shot 2022-10-26 at 3 38 28 PM](https://user-images.githubusercontent.com/19271145/198151883-301e14e0-5f97-407d-b285-36d28bf28a3c.png)

it works when viewing the issue directly in the repo
![Screen Shot 2022-10-26 at 3 40 32 PM](https://user-images.githubusercontent.com/19271145/198152183-7e334b6d-862e-4592-8c61-eefc494c4d78.png)
